features/integration/compile.feature

```diff
@@ -28,4 +28,4 @@ Feature: Compile
     Then the resulting source map is the same as the json "<sourcemap>"
     Examples:
       | teal                | sourcemap                                   |
-      | programs/quine.teal | v2algodclient_responsejsons/sourcemap.json  |
\ No newline at end of file
+      | programs/quine.teal | v2algodclient_responsejsons/sourcemap2.json |
```

Comment:
nit: is this correct? sourcemap2.json is deleted now

Reply:
annoying merge conflict ... thx for the catch

Reply:
wait, still not reverted...